draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -734,42 +779,48 @@ It is up to the Holder how to maintain the mapping between the Disclosures and t
 
 ## Processing by the Holder  {#holder_verification}
 
-If a Key Binding JWT is received by a Holder, the SD-JWT SHOULD be rejected.
+The Issuer MUST provide the Holder and SD-JWT, not an SD-JWT-KB.  If the Holder
+receives an SD-JWT-KB from the Issuer, it SHOULD be rejected.
 
 For presentation to a Verifier, the Holder MUST perform the following (or equivalent) steps:
 
  1. Decide which Disclosures to release to the Verifier, obtaining proper End-User consent if necessary.
- 2. Assemble the SD-JWT for presentation, including the Issuer-signed JWT and the selected Disclosures (see (#data_formats) for the format).
- 3. If Key Binding is required, create a Key Binding JWT ((#kb-jwt)) and add it to the SD-JWT for presentation.
- 4. Send the SD-JWT for presentation to the Verifier.
+ 2. Assemble the SD-JWT, including the Issuer-signed JWT and the selected Disclosures (see (#data_formats) for the format).
+ 3. If Key Binding is not required, send the SD-JWT to the Verifier.
```

Comment:
it's a little abrupt/disjoint that if this condition is met, send the SD-JWT to the Verifier and end, otherwise do the next steps